manifest.yml

```diff
@@ -3,12 +3,12 @@ name: Jupyter Lab
 category: Interactive Apps
 subcategory: Servers
 role: batch_connect
-description: "This app will launch a Jupyter Lab server on a Peregrine node.\r\n\r\n
+description: "This app will launch a Jupyter Lab server on a Habrok node.\r\n\r\n
     **IMPORTANT!**\r\n\r\n **Make sure you add the following line to your** `.bashrc` **file, otherwise it will not work:**\r\n\r\n
     ```unset XDG_RUNTIME_DIR```\r\n\r\n
     
     **Also, make sure you follow the steps detailed on the 
-    [wiki](https://wiki.hpc.rug.nl/peregrine/connecting_to_the_system/web_portal#jupyter_notebook) 
+    [wiki](https://wiki.hpc.rug.nl/habrok/connecting_to_the_system/web_portal#jupyter_notebook) 
     page for this app, to set up your environment.**"
-version: 0.2
+version: 0.1
```

Comment:
Is moving from 0.2 to 0.1 because this is a new application in Hábrok vs a given version in Peregrine?